README.md

```diff
@@ -1 +1,72 @@
 # ironfoudation-team-project
+# WOOF application 
+
+## Table of contents
+* [General info](#general-info)
+* [Technologies](#technologies)
+* [Installation](#installation)
+* [Sources](#sources)
+* [Authors](#authors)
+* [Important](#important)
+* [Notes](#notes)
+
+## General info
+
+This is the application for iOS.
+Woof application connects dog owners with pet sitters which contributes to build trusted relationships between them. The app makes it easy for nearby pet owners to find pet sitting and makes convienience of responding to jobs for pet sitters from phone.
+App also makes managing sitting bussines and pet owner's schedule. They can choose their own schedule, prices, and requirements for servise. 
+
+**Features include**
+-Secure store all of pet owners' and pet sitters' data base. 
+-Regularly data base update with personal information.
+
+**It also important to know:**
+To process application,some private cliets information like home adress, mobile phone is needed.
+
+## Technologies
+
+Project is created with:
+* macOS ventura 13.3
+* Xcode version: 14.3
+* Swift: 5.8
+* SDKs for IOS: 16.4
+* SwiftLint: 0.51.0
+* SwiftFormat: 0.51.7
+
+## Installation
+
+To run this project:
+```
+1)install it locally by cloning the current repository, using: git@github.com:ios-course/ironfoudation-team-project.git
+2)run the file Woof.xcodeproj
+```
```

Comment:
Maybe better to format the code like this, it's hard to read  this file:

To run this project:
1) install it locally by cloning the current repository via ssh:
```
 git@github.com:ios-course/ironfoudation-team-project.git
```
2) run the file Woof.xcodeproj

Reply:
LGTM, thanks for a good advice. Updated format.